_posts/2024-08-02-wayfinder-latest.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: postv2
+font: serif
+title: "WayFinder: The Evolution of our Recommendation Engine"
+description: "A journey through the complexity of creating a recommendation engine to foster high performing software teams"
+date: 2024-08-02
+highlight: monokai
+image: /img/wayfinder/recommendation-bar.png
+author: Hamish Tedeschi
+tags:
+  [
+    WayFinder,
+    DevOps,
+    DORA Metrics,
+    GitHub,
+    Gitlab,
+    Azure DevOps
+  ]
+---
+
+## Exploring the Latest Updates to WayFinder
+
+Our recent updates to [WayFinder](https://app.wayfinder.ninja) have ushered in significant changes, particularly with the development and refinement of its recommendation engine. As [WayFinder](https://app.wayfinder.ninja) pivots towards a more refined user experience, the new recommendation engine plays a central role in delivering tailored content and suggestions to users, based on the orgs Product state and the state of the Products (DORA) metrics, such as Lead Time or Deployment Frequency. It is more complex than originally thought, so I wanted to give you a taste of the journey..
```

Comment:
We can use just `wayfinder.ninja` there is a top level redirect to app.

Reply:
Agree on the redirect behaviour, but if this is the end result we might as well spell it out and potentially save a bunnyhop